src/containers/Footer/Footer.tsx

```diff
@@ -90,12 +96,12 @@ const Footer = () => {
             <div className="row pb-4">
               <div className="col navbar">
                 <ul className="navbar-nav fs-15">
-                  {renderFooterLeft}
+                  {RenderFooterItem(footerListLeft)}
```

Comment:
The `RenderFooterItem` is the react component. It should be `<RenderFooterItem footerList={footerListLeft} />`